src/main/java/duke/exception/EmptyTaskException.java

```diff
@@ -0,0 +1,14 @@
+package duke.exception;
+
+public class EmptyTaskException extends Exception {
+    private String taskType;
+
```

Comment:
Think you can javadoc here if you want LOL

Reply:
Damn, I can't believe I had missed out the point where all classes need JavaDoc headers!